index.html

```diff
@@ -207,6 +212,35 @@ <h3>Specific Verifiable Credentials</h3>
 that should work for most use cases.
         </p>
       </section>
+      <section>
+        <h3>Content Integrity</h3>
+        <p>
+Protecting the integrity of content is an important component of verification.
+<a>Verifiers</a> need to have confidence that the content they rely on to
+verify <a>credentials</a> doesn't change without their knowledge. This content
+may include data schemas, identifiers, public keys, etc.
+        </p>
+        <p>
+There are a number of ways to provide content integrity protection. A few of
+these are described in greater detail below.
+        </p>
+        <section>
+          <h4>Hashlinks</h4>
+
+          <p>
+Hashlink URLs can be used to provide content integrity for links to external
+resources.
+          </p>
+        </section>
+        <section>
+          <h4>Verifiable Data Registries</h4>
+          <p>
+A <a>verifiable data registry</a> can also provide content integrity protection.
+One example of a <a>verifiable data registry</a> which provides content
+integrity protection is a distributed ledger.
```

Comment:
I'm not a big fan of adding links to Wikipedia in specs, but we need a reference for distributed ledgers here too. (https://en.wikipedia.org/wiki/Distributed_ledger)

Reply:
I am going to add quite a bit more detail here. I will raise an issue to track that work so that this PR isn't hung up on it.

Reply:
I've added a comment to Issue #31 to track this

Reply:
👍